registry/workers-registry.js

```diff
@@ -0,0 +1,50 @@
+const { v4 : uuid } = require('uuid');
+const { pool } = require('../utils/db');
+
```

Comment:
For me, better was divide BLL and operations procedure with db

Reply:
I'm not pretty sure what is about, could you give me some more informations? :)

Reply:
Done